gtsam/hybrid/GaussianMixture.h

```diff
@@ -174,20 +174,44 @@ class GTSAM_EXPORT GaussianMixture
       const VectorValues &continuousValues) const;
 
   /**
-   * @brief Compute the logProbability of this Gaussian Mixture given the
-   * continuous values and a discrete assignment.
+   * @brief Compute the error of this Gaussian Mixture.
+   *
+   * This requires some care, as different mixture components may have
+   * different normalization constants. Let's consider p(x|y,m), where m is
```

Comment:
Yes, in the past I mental-modeled this as a discrete prior

Reply:
i think that is slightly different and is still useful. A "prior" would choose probabilistically between different components, even if they had the same normalization constant (which depends on the noise).

I would actually prefer to rename this class, `P(x|m)`, to GaussianComponent, and reserver GaussianMixture for `P(x|m)P(m)`.